pkg/topology/topology_loader.go

```diff
@@ -21,7 +22,8 @@ type containerTemplate struct {
 	ReqCPU float64
 	ReqMem float64
 
-	QPS target.Resource
+	QPS          target.Resource
+	ResponseTime *target.Resource
```

Comment:
As mentioned in file `pkg/target/types.go`, the value of `containerTemplate` can be shared by multiple Pods. It is safe to use struct instead of pointer.